paper/paper.tex

```diff
@@ -344,70 +422,81 @@ \subsubsection{OpenFOAM}
 Custom build systems require an disproportional amount of additional work to make them runnable on any system.
-OpenFOAM and Allwmake are prime examples of this issue. Package managers have to understand a whole new build system for a single package -- a fact that is often ignored when considering the trade-off between maintaining a custom build system versus pivoting to an industry standard, such as CMake or autotools.
+OpenFOAM and Allwmake are prime examples of this issue. Package managers have to understand a whole new build system for a single package -- a fact that is often ignored when considering the trade-off between maintaining a custom build system versus pivoting to an industry standard, such as CMake or Autotools.
 Such standard build tools generally provide interfaces for dependency management and optional features. This is often better than only documenting these, since documentation needs to be kept in sync with the underlying code.
 
 The authors of xSDK\footnote{\url{https://xsdk.info/}} come to a similar conclusion.
-They provide a list of package policies~\cite{xSDK2023}, which, for example, specify that packages must have an appropriate build system, which includes CMake and autoconf as examples.
+They provide a list of package policies~\cite{xSDK2023}, which, for example, specify that packages must have an appropriate build system, which includes CMake and Autotools as examples.
 If a package supports all policies, they can be added to the growing list of packages in xSDK.
 It is no surprise that xSDK packages (e.g., PETSc, deal.II, or preCICE) are also easy to package with Nix.
 
-The xSDK policies also requires portable installations. Several components in the preCICE ecosystem require installation into \texttt{\$HOME} or any required files to be located at specific locations. Both, the CalculiX-preCICE adapter and the SU2-preCICE adapter require the original source code to be present as they add new features on top or patch the source code.
-The current solution of cloning the code at the specific location works, but it might be better to look into Git submodules. This would moreover allow pinning the version of the original code compared to only documenting it.
+The xSDK policies also requires portable installations. Several components in the preCICE ecosystem require installation into \texttt{\$HOME} or any required files to be located at specific locations.
+Both, the CalculiX-preCICE adapter and the SU2-preCICE adapter require the original source code to be present as they add new features on top or patch the source code.
+The current solution of requiring to clone the code into a specific directory works, but it might be better to look into Git submodules, as this would result in a more portable solution.
+This would, moreover, allow pinning the version of the original code compared to only documenting it.
```

Comment:
Some other projects (e.g., I am currently reviewing a JOSS paper for GEOS: https://github.com/GEOS-DEV) use their own mirrors for third-party libraries / dependencies, for reproducibility / availability / security reasons.